db/migrate/20170303023308_create_exams.rb

```diff
@@ -0,0 +1,13 @@
+class CreateExams < ActiveRecord::Migration[5.0]
+  def change
+    create_table :exams do |t|
+      t.string :score
+      t.datetime :exam_date
```

Comment:
Just a date. I don't think it needs to be a datetime. I don't think students are taking multiple exams in one day on the same subject such that they need to be differentiated between the 10AM exam and the 2PM exam.